words/words.txt

```diff
@@ -0,0 +1,7 @@
+apple,
```

Comment:
I think the file can be split by newlines instead of comma.

```
apple
banana
moose
```

And so on. What do you think?